README.md

```diff
@@ -32,24 +32,31 @@ export const runtime = 'edge';
 2. **Set environment variables**
 
 ```sh
-export FLEEK_PROJECT_ID=<your project id>
 export FLEEK_PAT=<your personal access token>
```

Comment:
Let's please use FLEEK_TOKEN instead, to remain consistent with the fleek CLI, I changed it because of that and forgot to change it here